console/src/components/Services/EventTrigger/Add/AddTrigger.js

```diff
@@ -468,6 +513,36 @@ class AddTrigger extends Component {
       );
     });
 
+    const getManualInvocationOption = () => {
+      return (
+        this.props.tableName && (
+          <div className={styles.manualInvocationCheckbox}>
+            <label>
+              <input
+                className={`${styles.display_inline} ${styles.add_mar_right}`}
+                type="checkbox"
+                value={manualInvocation.name}
+                checked={manualInvocation.isChecked}
+                onChange={manualInvocation.onChange}
+                data-test={manualInvocation.testIdentifier}
+              />
+              Invoke this trigger via Data browser
```

Comment:
I think a better label would be `Allowing invoking this trigger via data browser`.
@tirumaraiselvan @karthikvt26 thoughts?

Reply:
@karthikvt26 Can we change the tooltip text to point to docs link?